### DISCUSSION TITLE
How to listen to scrollTo "cancel"

### DISCUSSION THREAD

**Discussion by quentinneyraud** [darkroomengineering/lenis#421] [Q&A]
Hi,

Is there a way to detect user scroll events while the scrollTo animation is running ? 
I am doing it with userData right now but there is maybe a better option ?  
https://codepen.io/Quentinneyraud/pen/wBwoKyX

**Comment by clementroche** [comment]
Currently there is no way to detect that besides using `userData` as you did. I could add a property such as `initiator` that could be either `wheel`, `touch` or `scrollTo` this would require some internal changes.
